cumulusci.yml

```diff
@@ -463,7 +463,9 @@ tasks:
                 if (!String.isEmpty(shouldScheduleJobs)) {
                     params.put('ScheduleJobs', Boolean.valueOf(shouldScheduleJobs));
                 }
-                Callable apiClass = new %%%NAMESPACE%%%Callable_Api();
+                String ns = ('%%%NAMESPACE%%%').replace('__','');
```

Comment:
@meena-sfdc FYI When testing the `enable_crlp` task last night in a managed package, I realized that the instantiation of the Callable_Api class in a managed package environment was not correct. I figured out the correct way to do this so that it works for both managed and unmanaged, and applied that change into this branch. See lines 466-468.

@RadSFDeveloper You'll need to apply these 3 lines to your PR as well where it instantiates the Callable_Api class. in the `deploy_rd2_testing` flow.